src/main/java/org/gwtproject/tools/cldr/LocaleData.java

```diff
@@ -15,1117 +15,1276 @@
-    maps.clear();
-  }
-
-  /**
-   * Summarize values known by territories to bare languages, based on
-   * populations using a particular value.
-   * 
-   * @param category
-   * @param key
-   * @param values map of region codes to values
-   */
-  public void summarizeTerritoryEntries(String category, RegionLanguageData regionLanguageData,
-      String key, Map<String, String> values) {
-    for (GwtLocale locale : allLocales.keySet()) {
-      if (locale.getRegion() != null || locale.getLanguage() == null) {
-        // skip any that have a region or don't have a language
-        continue;
-      }
-      String language = locale.getAsString();
-      Map<String, Double> valueMap = new HashMap<String, Double>();
-      for (RegionLanguageData.RegionPopulation langData : regionLanguageData.getRegions(language)) {
-        String region = langData.getRegion();
-        GwtLocale regionLocale = localeFactory.fromString(language + "_" + region);
-        String value = getTerritoryData(regionLocale, values);
-        if (value != null) {
-          Double pop = valueMap.get(value);
-          if (pop == null) {
-            pop = 0.0;
-          }
-          pop += langData.getLiteratePopulation();
-          valueMap.put(value, pop);
-        }
-      }
-      double max = 0;
-      String maxValue = null;
-      for (Map.Entry<String, Double> entry : valueMap.entrySet()) {
-        if (entry.getValue() > max) {
-          max = entry.getValue();
-          maxValue = entry.getKey();
-        }
-      }
-      if (maxValue != null) {
-        addEntry(category, locale, key, maxValue);
-      }
+
+    /**
+     * Summarize values known by territories to bare languages, based on
+     * populations using a particular value.
+     *
+     * @param category
+     * @param key
+     * @param values   map of region codes to values
+     */
+    public void summarizeTerritoryEntries(String category, RegionLanguageData regionLanguageData,
+                                          String key, Map<String, String> values) {
+        for (GwtLocale locale : allLocales.keySet()) {
+            LOGGER.info("Summarize territory entries for locale : "+locale);
+            if (locale.getRegion() != null || locale.getLanguage() == null) {
+                // skip any that have a region or don't have a language
+                continue;
+            }
+            String language = locale.getAsString();
+            Map<String, Double> valueMap = new HashMap<String, Double>();
+            for (RegionLanguageData.RegionPopulation langData : regionLanguageData.getRegions(language)) {
+                String region = langData.getRegion();
+                GwtLocale regionLocale = localeFactory.fromString(language + "_" + region);
+                String value = getTerritoryData(regionLocale, values);
+                if (value != null) {
+                    Double pop = valueMap.get(value);
+                    if (pop == null) {
+ 
```

Comment:
seems weird to append to a string... which you give to a stringbuilder. not your fault though, just weird

Reply:
this whole method should just be += if we're already mixing and matching :p